template/module/tests/test_something.py

```diff
@@ -26,3 +26,18 @@ def test_something(self):
         Any method starting with ``test_`` will be tested.
         """
         pass
+
+
+class UICase(HttpCase):
```

Comment:
Yes and not. HttpCase has been around for awhile, but the code to execute was different before (because it's js, that it has been changed).